examples/scripts/deconv_ppp_bm4d_admm.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# This file is part of the SCICO package. Details of the copyright
+# and user license can be found in the 'LICENSE.txt' file distributed
+# with the package.
+
+"""
+Image Deconvolution (ADMM Plug-and-Play Priors w/ BM4D)
+=======================================================
+
+This example demonstrates the use of class
```

Comment:
A bit more detail would be helpful here. At the very least, mention that the problem is 3D.